### ISSUE TITLE
Version 3.13 and Version 3.12 are the same...?

### ISSUE THREAD

**Issue by TGirlChu** [stewienj/SwordfishCollections#37]
I am trying out the fix on #35 with the latest nuget package but the issue seems to be still reoccurring. I checked the Release note of the latest version (3.13) and realized the released assets are identical with 3.12 from Nov 2022: https://github.com/stewienj/SwordfishCollections/compare/3.12...3.13

Can you confirm if the update has been properly released in the nuget? Thanks in advance.

**Comment by stewienj** [comment]
oh no, my pull request didn't merge. Master branch is still behind the issue 35 branch.

**Comment by stewienj** [comment]
This release action is running now. https://github.com/stewienj/SwordfishCollections/blob/master/.github/workflows/release.yml
Should be complete in a few minutes.

**Comment by stewienj** [comment]
New release here https://www.nuget.org/packages/Swordfish.NET.CollectionsV3/3.3.14

**Comment by stewienj** [comment]
I was curious about why the 3.3.13 release didn't require me to update System.Collections.Immutable to version 9.0.0. I'm wondering if I should have left that at version 5.0.0 as I don't need any newer version than 5.0.0.

**Comment by stewienj** [comment]
Yep, I should have left the minimum required version of System.Collections.Immutable at 5.0.0. I've got a custom code analyzer that won't run because the Visual Studio compiler has System.Collections.Immutable version 8.0.0 in memory and it can't load version 9.0.0 specified by this library.
